docs/_attic/glossary_entries.py

```diff
@@ -193,6 +193,13 @@
 	institute="", 
 	pronunciation="")
 
+DRS = GlossEntry("DRS",
+        acronym_full="Data Repository Service",
+        definition="A standardized [API], created by the [GA4GH] Cloud Work Stream, that provides portable access to a repository of data resources.",
```

Comment:
I would say "to repositories of data resources".

And/or, from https://ga4gh.github.io/data-repository-service-schemas/preview/release/drs-1.0.0/docs/, "a generic interface to data repositories".